### PR TITLE
Close #5671: Update component status in README

### LINKED ISSUE #5671: Evaluate the component status in the README
Our [README](https://github.com/mozilla-mobile/android-components/blob/master/README.md) shows the components and their status. Since shipping a-c 1.0, we regarded that as the minimum requirements for shipping a browser.

We should re-evaluate the status icons on the README to see how we feel about moving them from 🔴 -> ⚪️ -> 🔵 .

### PR DESCRIPTION
**Pull request by jonalmeida**
---

Close #5671

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

**Comment by jonalmeida** [comment]
Let's leave this open for a day or so in case others want to have a peek or if something is not accurately reflected.

File: README.md [hunk 1]
```diff
@@ -66,111 +66,111 @@ The API reference docs are available at [mozac.org/api/](https://mozac.org/api/)
...
-* 🔴 [**Intent**](components/feature/intent/README.md) - A component that provides intent processing functionality by combining various other feature modules.
+* 🔵 [**Intent**](components/feature/intent/README.md) - A component that provides intent processing functionality by combining various other feature modules.
 
-* 🔴 [**Progressive Web Apps (PWA)**](components/feature/pwa/README.md) - A component that provides functionality for supporting Progressive Web Apps (PWA).
+* ⚪ [**Progressive Web Apps (PWA)**](components/feature/pwa/README.md) - A component that provides functionality for supporting Progressive Web Apps (PWA).
 
 * 🔴 [**P2P**](components/feature/p2p/README.md) - A component that provides peer-to-peer communication.
 
-* 🔴 [**Reader View**](components/feature/readerview/README.md) - A component that provides Reader View functionality.
+* 🔵 [**Reader View**](components/feature/readerview/README.md) - A component that provides Reader View functionality.
```
Comment (csadilek) [inline README.md:139]: I would vote for "Preview" state here. I mean it works well, but we have some short comings relative to Fennec until we have time to do the last big refactor (extension page).

**Comment by jonalmeida** [comment]
bors r=Amejia481,csadilek

**Comment by Amejia481** [comment]
bors retry

**Pull request merged**